### PR TITLE
chore: Metrics for Base Executor

### PR DESCRIPTION
**Pull request by hendrikebbers**
Fixes https://github.com/hashgraph/hedera-services/issues/12873

### CODE REVIEW

**Review by mxtartaglia-sl — APPROVED** [review]

**Review by cody-littley — APPROVED** [review]
I'm marked as a code owner, but I don't actually own any code in this PR. Approving in case that makes the PR process easier. Please seek reviews from other code owners before merging.

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**
